labs.yml

```diff
@@ -21,6 +21,9 @@ commands:
       - name: skip-validation
         default: true
         description: Validate Transpiled Code, default True validation skipped, False validate
```

Comment:
add `warehouse_types: ["SERVERLESS", "PRO"]` in the `installer` section to get warehouse prompt. see:
- https://github.com/databricks/cli/blob/36241ee55e9708fe5519c330e102174031016aea/cmd/labs/project/entrypoint.go#L49-L54
- https://github.com/databricks/cli/blob/36241ee55e9708fe5519c330e102174031016aea/cmd/labs/project/entrypoint.go#L242-L244
- https://github.com/databricks/cli/blob/36241ee55e9708fe5519c330e102174031016aea/cmd/labs/project/installer.go#L35